package.json

```diff
@@ -22,13 +22,17 @@
     "vdom"
   ],
   "scripts": {
-    "test": "jest --coverage --no-cache && tsc -p test/ts",
-    "build": "npm run bundle && npm run minify",
+    "test": "jest --coverage --no-cache",
+    "posttest": "tsc -p test/ts",
+    "build": "npm run bundle",
+    "postbuild": "npm run minify",
```

Comment:
Why do you need `build` and `postbuild` if you can run `bundle` and `minify` tasks directly?

Reply:
Explanation is in the main comment.

Reply:
Your example:
```sh
in: npm run build
out: npm run bundle
out: rollup -i src/index.js -o dist/hyperapp.js -m -f umd -n hyperapp
out: npm run minify
out: uglifyjs dist/hyperapp.js -o dist/hyperapp.js -mc pure_funcs=['Object.defineProperty'] --source-map includeSources,url=hyperapp.js.map
```
Behavior which I see:
```sh
in: npm run build
out: npm run bundle
out: rollup -i src/index.js -o dist/hyperapp.js -m -f umd -n hyperapp
```
```sh
in: npm run bundle
out: rollup -i src/index.js -o dist/hyperapp.js -m -f umd -n hyperapp
```
tasks `build` and `bundle` works the absolutely the same. Why do you need both of them?

Reply:
Do I understand correctly that npm runs three commands when you run only one?
For example when you type `npm run build`, npm will execute these ones with given the order:
- `npm run prebuild`
- `npm run build`
- `npm run postbuild`